ui/app/styles/components/stat-text.scss

```diff
@@ -0,0 +1,100 @@
+.stat-text-container {
+  line-height: normal;
+
+  &.l {
+    > .stat-label {
+      font-size: $size-5;
+      font-weight: $font-weight-semibold;
+      line-height: inherit;
+    }
+    .stat-text {
```

Comment:
curious here why you didn't need the `>` but you have it down below on row 23? Does this work without the carrot, e.g. is it not nested?